plugins/single_plugins/center-view.cpp

```diff
@@ -0,0 +1,36 @@
+#include <output.hpp>
+#include <core.hpp>
```

Comment:
do you need that include?

Reply:
Probably not, but what provides Wayfire_plugin_t?

Reply:
plugin.hpp

Reply:
@ammen99 How is it working then? I'm not including that